.config/playwright.config.js

```diff
@@ -5,5 +5,6 @@ export default {
 	},
 	testMatch: /.*.spec.ts/,
 	testDir: "..",
-	globalSetup: "./playwright-setup.js"
+	globalSetup: "./playwright-setup.js",
+	workers: 2
```

Comment:
If this is flake in CI, it only ever runs with 1 worker anyway due to the available CPU cores, we could explicitly set this to 1 though to make it consistent with CI but they run much faster locally if we just remove this (because it will run on multiple cores locally).

Reply:
hmm yea it's just not showing up - maybe the toast is closed by then because of the timer? But why would that be 🤔 

<img width="1269" alt="image" src="https://github.com/gradio-app/gradio/assets/41651716/f6739d88-fe11-4299-b528-42be0dd2f834">

Reply:
The timer is like 10 seconds, tho. Feels like a long enough time. I wonder if there is a way to capture logs or something. Maybe there is some weird error that we aren't thinking about.

Reply:
> My bad - I thought the GH runner had 2 cpu cores.

And you are correct but playwright seems to refuse to use them both, i think it won't use more than 50% or something.  Weird anyway.